sdk/tests/conformance2/textures/misc/tex-storage-compressed-formats.html

```diff
@@ -87,15 +87,28 @@
         var tex = gl.createTexture();
         gl.bindTexture(gl.TEXTURE_2D, tex);
         gl.texStorage2D(gl.TEXTURE_2D, 1, internalformat, 1, 1);
-        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texStorage2D should succeed for " + enumToString(internalformat));
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+                            "texStorage2D should succeed for " + enumToString(internalformat));
         gl.deleteTexture(tex);
 
-        // Test a 3D texture.
+        // GLES 3.0.4 p147:
+        // "If internalformat is an ETC2/EAC format, CompressedTexImage3D will generate an
+        //  INVALID_OPERATION error if target is not TEXTURE_2D_ARRAY."
```

Comment:
The base OpenGL ES 3.0 specification doesn't define any more compressed texture formats than these, but extensions like https://www.opengl.org/registry/specs/KHR/texture_compression_astc_hdr.txt can add more.